src/main/resources/python/generate_stubs.py

```
@@ -287,10 +294,25 @@ def generate_bpy_types():
         bases = [b.__name__ for b in cls.__bases__ if b is not object]
         # Preserve order while deduping
         bases = list(dict.fromkeys(bases))
+
+        dependencies = set()
+        if hasattr(cls, "bl_rna"):
+            for prop in cls.bl_rna.properties:
+                if prop.identifier == "rna_type": continue
```

Ruff (0.14.10)

Multiple statements on one line (colon)

(E701)